pkg/apps/application-list.jsx

```diff
@@ -169,6 +196,9 @@ export const ApplicationList = ({ metainfo_db, appProgress, appProgressTitle, ac
             {comps.length == 0
                 ? <EmptyStatePanel title={ _("No applications installed or available.") } />
                 : <PageSection>
+                    {!progress && dataPackagesInstalled == false && <Alert variant="warning"
+                            isInline
+                            title={_("Appstream data package is missing, only installed applications are shown.")} />}
```

Comment:
Please align the properties. Probably best to move the `<Alert>` to the next line (possibly all props fit on one line then).